### DISCUSSION TITLE
Option to add a custom loader while CSV is being uploaded

### DISCUSSION THREAD

**Discussion by wouter-muller** [jgile/vue-csv-import#73] [Q&A] [answered]
I am using vue-csv-import in a project and really love it and appreciate the effort that went into building it!

I was wondering about one thing though, how can i add a custom loader indication (some kind of spinner or animation) that is visible while the .csv file is being uploaded? Is there some data property that i could watch in the component where i use vue-csv-import? I think this would be helpful for users to understand that something is happening.

Thanks in advance!

**Comment by James-Burgess** [comment]
The object you pass as v-model, in the [example](https://github.com/jgile/vue-csv-import/blob/30ef278a014ef89dc62a09a1974a75312115da3e/src/App.vue#L30) `csv` is populated once a file has been uploaded.

**Comment by jgile** [comment, marked as answer]
as of v4.1.2 you can pass an axios config object to vue-csv-submit via the config prop.  you should be able to use onUploadProgress to get upload progress on submit.
